Pod/Classes/UITextView+URBNHighlightExtra.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UITextView+URBNHighlightExtra.h
```

Comment:
Speaking of naming, the `extra` bit is not needed. `+URBNHighlight` should suffice